Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -75,8 +177,58 @@ public static int[] InsertFirst(int[] array, int number)
         /// <returns>A new array with element added in the end of array. If an array is empty or null, returns new array with number in it.</returns>
         public static int[] InsertLast(int[] array, int number)
         {
-            // ToDo: implement.
-            return array;
+            if (array == null)
+            {
+                return AddOneNumberToNullArray(number);
+            }
+
+            var lengthenedArray = MakeALongerArray(array);
+            
+            //place the new number at the end of the new, long array
+            PlaceTheNewNumberAtTheAppropriateIndexPosition(lengthenedArray, "Last", number);
+            
+            //repopulate the rest of the numbers at their same index in the longer array
+            for (int i = 1; i < lengthenedArray.Length-1; i++)
+            {
+                lengthenedArray[i] = array[i];
+            }
+            return lengthenedArray;
+            
+        }
+
+        private static void PlaceTheNewNumberAtTheAppropriateIndexPosition(int[] lengthenedArray, string position, int number)
+        {
+            switch (position)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Replace this 'switch' statement with 'if' statements to increase readability.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4887772)